### PR TITLE
This commit update docker push into quay.io

### PR DESCRIPTION
**Pull request by phracek**
CentOS7 images are now after merge
pushed into Docker.io and Quay.io

What needs to be reviewed?
* configuration
* yaml/builders/image-build-push.yaml 

The rest files are generated from `template`

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: yaml/builders/image-build-push.yaml [hunk 1]
```diff
@@ -32,17 +32,23 @@
...
               esac
 
               # By default images are now tagged into localhost/ registry
-              image_name=${{image/#localhost\//}}
-              image_name=quay.io/${{image_name}}
-              echo "Pushing ${{image_name}}"
+              # Image name for Docker Hub
+              # hub_namespace is docker.io/centos and we need to get from image only image name
+              image_name={hub_namespace}/\${{image##*/}}
+              echo "Pushing ${{image}} into Quay Hub"
```
Comment (hhorak) [inline yaml/builders/image-build-push.yaml:47]: Isn't there supposed to be a `docker tag` call before `docker push` as well?

**Comment by hhorak** [comment]
I didn't test, but the changes look sane. There is just a single question about missing `docker tag` but otherwise LGTM.

File: yaml/builders/image-build-push.yaml (hunk 1 above)
Comment (phracek) [reply yaml/builders/image-build-push.yaml:47]: Good point. By default, all our images are built like `quay.io/centos7/<image_name>`. Before this change, images were built with tag `centos/<image_name>`. Therefore docker tag was needed.

Comment (hhorak) [reply yaml/builders/image-build-push.yaml:47]: Thanks, makes sense.

**Pull request merged**